src/atomic/file.rs

```diff
@@ -105,7 +107,8 @@ impl AtomicFile {
         // This UID must be treated as confidential information.
         // Depending on network transport used to sync the files (if any),
         // it can leak to an unauthorized party.
-        let machine_id = machine_uid::get()?;
+        let machine_id = app_id::read()?;
```

Comment:
We don't have to keep this term "machine id", we can call the variable something like `local_peer` to reflect the purpose of having it.